packages/core/src/base-axis-chart.ts

```diff
@@ -136,6 +142,107 @@ export class BaseAxisChart extends BaseChart {
 		}));
 	}
 
+	createSlider(){
+
+		//TODO get the name of the action bar for the chart
+		var divToInsert = "#actions-" + this.options.type
+		console.log(divToInsert)
+
+		var handle_radius = 50,
+			handle_start_val = 50,
+			handle_padding = 10,
+			slider_height = 5,
+			axis_height = 20;
+
+		//insert the slider in the action bar element
+		var setup = marcon()
+				.width(window.innerWidth)
+				.height(handle_radius * 2 + slider_height + handle_padding + axis_height)
+				.left(handle_radius)
+				.right(handle_radius)
+				.bottom(axis_height)
+				.element(divToInsert);
```

Comment:
We should include some comments here as to what some of these lines mean, and what they do